templates/default/rt/rtqueueinfo.html

```diff
@@ -236,13 +236,13 @@ <H1>{$layout.pagetitle}</H1>
 	{/if}
 	<TR>
 	    <TD COLSPAN="3" class="text-right">
-	        <A HREF="?m=rtqueueview&id={$queue.id}"><IMG src="img/view.gif" alt=""> {trans("Browse")}</A>
-	    	<A HREF="?m=rtqueueedit&id={$queue.id}"><IMG src="img/save.gif" alt=""> {trans("Edit")}</A>
+		{button icon="view" href="?m=rtqueueview&id={$queue.id}" label="Browse"}
+		{button icon="save" href="?m=rtqueueedit&id={$queue.id}" label="Edit"}
 		{if $queue.deleted == 1 && $queue.deltime != 0 && (ConfigHelper::checkPrivilege('helpdesk_advanced_operations'))}
-			<A id="restore-rtqueue" HREF="?m=rtrestore&id={$queue.id}&qaction=restore" data-name="{$queue.name}"><IMG src="img/reload.gif" alt="{trans("Restore<!RT>")}" title="{trans("Restore<!RT>")}"> {trans("Restore<!RT>")}</A>
-			<A id="remove-rtqueue" HREF="?m=rtremove&id={$queue.id}&qaction=delperm" data-name="{$queue.name}"><IMG src="img/delete.gif" alt="{trans("Delete permanently")}" title="{trans("Delete permanently")}"> {trans("Delete permanently")}</A>
+			{button id="restore-rtqueue" href="?m=rtrestore&id={$queue.id}&qaction=restore" data_name="{$queue.name}" icon="restore" label="Restore<!RT>"}
+			{button id="remove-rtqueue" href="?m=rtremove&id={$queue.id}&qaction=delperm" data_name="{$queue.name}" icon="delete" label="Delete permanently"}
 		{else}
```

Comment:
Z tego widzę, że Ty raczej nie testujesz w ogóle tego co robisz - to sensownie nie zadziała po przekształceniu `<a>` na `{button}`, gdyż robisz href, a tu chodzi o to, że na dole masz kod JS, który realizuje potwierdzenie operacji. Popatrz jak to się zachowa...

Reply:
Wyciągasz pochopne wnioski - nie da się wydajnie testować swojego kodu. Nie zauważyłem tego po prostu.

Może warto dorobić do `{button` parametr  `confirmation_text=` a jeśli go uzupełnimy to się zadzieje to co jest aktualnie w JS ?

Reply:
@chilek rozumiesz o co mi chodzi? Spójrz na ilość wystąpień "confirmationDialog".

Reply:
Na tą chwilę cofnąłem tą zmianę https://github.com/lmsgit/lms/pull/2017/commits/2ef1e48e101178a193ea66e0847d5e0e9b77e0fc po to byś mógł puścić PR.